src/client.rs

```diff
@@ -167,7 +195,7 @@ impl InstrumentedClientRequest {
                 )
             )
             .with_kind(SpanKind::Client)
-            .with_attributes(attributes)
+            .with_attributes(self.attrs.clone())
```

Comment:
this could use [mem::take](https://doc.rust-lang.org/std/mem/fn.take.html) instead to avoid cloning the vec

Reply:
updated